packages/regl-worldview/src/commands/GLText.js

```diff
@@ -142,7 +142,7 @@ const vert = `
   attribute vec2 srcOffset;
   attribute float srcWidth;
   attribute vec2 destOffset;
-
+  
```

Comment:
#nit unnecessary whitespace change. I have my editor set up to remove trailing whitespaces from each line on save; I'd recommend this to everyone for consistency. 😄

Reply:
Thanks JP! I already setup vscode to remove trailing whitespaces from now on